x-pack/plugins/code/public/components/file_tree/props.json

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
Can we put this and other `props.json` files inside a `__fixtures__` directory so it's a little more clear what they're for?